compatibility.h

```diff
@@ -0,0 +1,134 @@
+/* -------------------------------------------------------------------------
+ *
+ * compatibility.h
+ *
+ * Definitions for maintaining compatibility across Postgres versions.
+ *
+ * Copyright (c) 2010-2020, PostgreSQL Global Development Group
+ *
+ * -------------------------------------------------------------------------
+ */
+
+#ifndef SET_USER_COMPAT_H
+#define SET_USER_COMPAT_H
+
+/*
+ * PostgreSQL version 13+
+ *
+ * Introduces QueryCompletion struct
+ */
+#if PG_VERSION_NUM >= 130000
+static void PU_hook(PlannedStmt *pstmt, const char *queryString,
+					ProcessUtilityContext context, ParamListInfo params,
+					QueryEnvironment *queryEnv,
+					DestReceiver *dest, QueryCompletion *qc);
+#define _PU_HOOK \
+static void PU_hook(PlannedStmt *pstmt, const char *queryString, \
```

Comment:
can you introduce a tab here to be consistent with the other macro definitions? Same applies to the rest of the macros below. It increases readability imo